lua/src/lib/library.lua

```diff
@@ -1,9 +1,6 @@
 local M = { }
 local escape_pattern
-do
-  local _obj_0 = require("lapis.util")
-  escape_pattern = _obj_0.escape_pattern
-end
+escape_pattern = require("lapis.util").escape_pattern
```

Comment:
it appears that the .moon and .lua files have different changes. Am I missing something? I have a git hook setup in my env to ensure this is never the case.